frontend/src/views/SignIn.vue

```diff
@@ -2,7 +2,7 @@
   <div class="sign-in" v-if="active">
     <h1>Sign in to Cinema Shop</h1>
     <h2>Please enter your sign in details.
-    <a href="#/sign-up">Sign up</a>
+    <a href="/#sign-up">Sign up</a>
```

Comment:
I experienced problems with sign-up and tried to figure it out. Now I guess that should be like **"<a href="#/sign-up">Sign up</a>"** because **/#** is used for scrolling page to certain item and that's not the thing we need in this task.

Reply:
Where are genre, actor, cinema hall changes? I am not sure that this functionality was intended at all... We have the button "add" and it works, but I can't even find a button in given frontend views similar to "change" or "edit" neither actors nor movies, etc...